tests/integration/pilot/common/routing.go

```diff
@@ -2904,6 +2905,103 @@ func VMTestCases(vms echo.Instances) func(t TrafficContext) {
 	}
 }
 
+func TestExternalService(t TrafficContext) {
+	// Let us enable outboundTrafficPolicy REGISTRY_ONLY
+	// on one of the workloads, to verify selective external connectivity
+	SidecarScope := fmt.Sprintf(`apiVersion: networking.istio.io/v1alpha3
+kind: Sidecar
+metadata:
+  name: restrict-external-service
+  namespace: %s
+spec:
+  workloadSelector:
+    labels:
+      app: a
+  outboundTrafficPolicy:
+    mode: "REGISTRY_ONLY"
+`, t.Apps.EchoNamespace.Namespace.Name())
+
+	fakeExternalAddress := t.Apps.External.All[0].Address()
+	parsedIP := net.ParseIP(fakeExternalAddress)
```

Comment:
Hi @kfaseela, suggest that change this from `net.ParseIP` to `netip.ParseAddr` because of issue: 
[Adopt go 1.18 net/netip package](https://github.com/istio/istio/issues/39348).  And another package `net/netip` should be added in head of this file.

Reply:
@zhlsunshine changed to use netip.ParseAddr()